internal/async/wait.go

```diff
@@ -84,6 +85,6 @@ func WaitSync(config *WaitSyncConfig) (terminalValue interface{}, err error) {
 		return nil, err
 	case <-time.After(config.Timeout):
 		timeout <- true
-		return nil, fmt.Errorf("timeout after %v", config.Timeout)
+		return nil, errors.New("timeout after %v", config.Timeout)
```

Comment:
We say that we should return internal error only on public method to avoid double wrapping.